frontend/src/app/register-form/register-form.component.html

```diff
@@ -0,0 +1,46 @@
+<mat-card class="register-card">
+  <mat-card-header class="register-card__header">
+    <div mat-card-avatar class="register-card__header__icon"></div>
+    <mat-card-title>Register</mat-card-title>
+  </mat-card-header>
+  <mat-card-content>
+    <form
+      class="register-card__form"
```

Comment:
I think adding `register-card` prefix to all CSS classes is not necessary because Angular do CSS shadowing underneath. This means that component stylesheets are isolated from each other.

Reply:
Yeah, I know about shadowing feature. I decided to use [BEM](http://getbem.com/introduction/) approach to make style files prettier. Should I remove it and apply your suggestion?

Reply:
I'll leave the final decision to you ;)

Reply:
I'will stick to my solution, I like how the style file looks now